lib/Models/SdmxJsonCatalogItem.js

```diff
@@ -0,0 +1,931 @@
+    // _concepts (via concepts) is both set and read in rebuildData.
+    // A solution to this would be to make concepts a Promise, but that would require changing the UI side.
+    knockout.track(this, ['_concepts']);
+
+    overrideProperty(this, 'concepts', {
+        get: function() {
+            return this._concepts;
+        }
+    });
+
+    knockout.defineProperty(this, 'activeConcepts', {
+        get: function() {
+            if (defined(this._concepts) && this._concepts.length > 0) {
+                return this._concepts.map(function(parent) {
+                    return parent.items.filter(function(concept) { return concept.isActive; });
+                });
+            }
+        }
+    });
+
+    knockout.getObservable(this, 'activeConcepts').subscribe(function() {
+        if (!this.isLoading) {
+            changedActiveItems(this);
+        }
+    }, this);
+
+    // knockout.getObservable(this, 'displayPercent').subscribe(rebuildData.bind(null, this), this);
+
+};
+
+inherit(CsvCatalogItem, SdmxJsonCatalogItem);
+
+defineProperties(SdmxJsonCatalogItem.prototype, {
+    /**
+     * Gets the type of data member represented by this instance.
+     * @memberOf SdmxJsonCatalogItem.prototype
+     * @type {String}
+     */
+    type: {
+        get: function() {
+            return 'sdmx-json';
+        }
+    },
+
+    /**
+     * Gets a human-readable name for this type of data source, 'GPX'.
+     * @memberOf SdmxJsonCatalogItem.prototype
+     * @type {String}
+     */
+    typeName: {
+        get: function() {
+            return 'SDMX JSON';
+        }
+    },
+
+    /**
+     * Gets the set of names of the properties to be serialized for this object for a share link.
+     * @memberOf ImageryLayerCatalogItem.prototype
+     * @type {String[]}
+     */
+    propertiesForSharing: {
+        get: function() {
+            return SdmxJsonCatalogItem.defaultPropertiesForSharing;
+        }
+    },
+
+    /**
+     * Gets the set of functions used to serialize individual properties in {@link CatalogMember#serializeToJson}.
+     * When a property name on the model matches the name of a property in the serializers object lieral,
+     * the value will be called as a function and passed a reference to the model, a reference to the destination
+     * JSON object literal, and the name of the property.
+     * @memberOf SdmxJsonCatalogItem.prototype
+     * @type {Object}
+     */
+    serializers: {
+        get: function() {
+            return SdmxJsonCatalogItem.defaultSerializers;
+        }
+    }
+});
+
+/**
+ * Gets or sets the default set of properties that are serialized when serializing a {@link CatalogItem}-derived for a
+ * share link.
+ * @type {String[]}
+ */
+SdmxJsonCatalogItem.defaultPropertiesForSharing = clone(CsvCatalogItem.defaultPropertiesForSharing); // TODO: do we need to add dataUrlComponent etc to this list?
+SdmxJsonCatalogItem.defaultPropertiesForSharing.push('regionDimensionId');
```

Comment:
The rule of thumb for `defaultPropertiesForSharing` is this: if it can be changed via the UI, it goes in `defaultPropertiesForSharing`, otherwise it doesn't.  So most of these things don't need to be included when sharing, right?